### ISSUE TITLE
Raspbian (Raspberry pie 3 install), example.py issue, ImportError: No module named instapy

### ISSUE THREAD

**Issue by 7ruth** [InstaPy/InstaPy#450]
Hello, 

I was able to run pip install . and all seemed fine.
Set up the example.py file for a first test run and got the above error after running sudo python examples/example.py from instaPy root. (using python 2.7 and pip)

I used pip freeze to see list of installed modules:
`
clarifai==2.0.20
configparser==3.5.0
EasyProcess==0.2.3
emoji==0.4.5
functools32==3.2.3.post2
future==0.15.2
instagram-py==0.0.1
jsonschema==2.5.1
Pillow==2.9.0
PyVirtualDisplay==0.2.1
requests==2.13.0
selenium==2.53.6`

I saw that module name was instagram-py, so I tried changing the import statement to this:
InstaPy = __import__("instagram-py") to no avail. 

I don't have python experience and I am sure this is something rudimentary, could someone point me in the right direction? Thank you. 

Edit... tried a few things that didn't help so far:
1. maybe python path was off, so added
'import sys
sys.path.append('/path/to/py_files_and_packages')'
at the top of my file, no go.
2. Checked permissions on the packages
instagram_py-0.0.1-py2.7.egg inside python2.7/dist-packages was set at -rw-r--r-- and I set it to 777, which still didn't help

**Comment by gitpatrickhub** [comment]
Just curious @7ruth, does the app run if you use the quickstart.py file for execution rather than the example.py file?

**Comment by 7ruth** [comment]
Thank you for a quick response @gitpatrickhub , in the old iteration of the repo I remember there was a quickstart.py in the root, but thats no longer the case.

I see there is a InstaPy/docker_conf/all_in_one/quickstart.py
as well as a docker_quickstart.py in the root. 

Are you referring to ether of these files? I will try them in a few hours when I get home and update here, but wanted to clarify. Thank you!

**Comment by gitpatrickhub** [comment]
@7ruth I'm running on rpi3, but not the Docker configuration. The file in: "InstaPy/docker_conf/all_in_one/quickstart.py" is the file I'm referring to, but if you are using Docker this likely will not work.

**Comment by 7ruth** [comment]
@gitpatrickhub , got it, thank you for the help. Unfortunately running that file resulted in the same error. 

Just to make sure I am doing the right thing:
Got exagear desktop, and installed chrome, works fine.
Got python 2.7.
Confirmed that my PY command path is set up.
(ran python -c "import sys; print(':'.join(x for x in sys.path if x))") and that seemed fine (lists instagram_py-0.0.1-py2.7.egg).
Git clone the latest repo, then python setup.py install, looked like it went without errors.
Installed selenium and pyvirtual which were also listed as paths in the step when I checked PY paths.
Also got chromedrive and put it in assets.
Then set up my example file and ran it, when I got the error.
Seems the issue is more python itself then other dependencies since it can't find that module for some reason.

Am I missing something else that you had to do? Thank you for your input.

**Comment by gitpatrickhub** [comment]
@7ruth I wasn't able to get Exagear to work with Chrome. So ditched Exagear and am running it on Firefox. Did you run the setup.py from within the Exagear session? Only other thing I can think of is to double check the case of "InstaPy" and make sure it matches in the module. 

The steps I took to run this on the rpi3 are included in the Docs folder, 'How to on Raspberry'

**Comment by 7ruth** [comment]
Hey @gitpatrickhub , thank you for the help, you pointed me in the right direction and I got it to work. 

Might help anyone else:
No exagear and no chrome (I started off from Tim Grossman's blog post that had a link to that)
I reinstalled everything following directions in the docs/How_to_Raspberry.md
Note, dont use docker_cond/all_in_one/firefoxExample.py but instead example/firefoxExample.py (shows different settings necessary to run it on Rasp Pie 3)
I tried to run the command (sudo xvfb-run python quickstart.py), after all other install steps, from the InstaPy/example director, I once more got the same error as before. 
I entered python shell and used help('modules') to see what was installed, instapy wasn't listed.
I cd .. to /InstaPy and ran the same command, 'instapy' was now in the help('modules') list.
It worked once I used sudo xvfb-run python example/fireFoxExample.py (also don't forget to put your credentials in the firefoxExample.py)
